org.civicrm.reqangular/test/decorators/ui-bootstrap/uib-modal.decorator.spec.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'common/lodash',
+  'common/angularMocks',
+  'common/angular',
+  'common/decorators/ui-bootstrap/uib-modal.decorator',
+  'common/angularBootstrap'
+], function (_, angularMocks, angular, $uibModalDecorator, $document) {
```

Comment:
First you list the modules that you need to access in the callback, then the others. There is no use for `angularMocks`, so it shouldn't be there

Reply:
AngularMocks define the `module` function.